src/lib/soy-ast.ts

```diff
@@ -177,30 +177,35 @@ export class IfCommand extends Command {
   condition: Expression;
   whenTrue: Command[];
   whenFalse: Command[];
+  private separator: string;
 
   constructor(
     condition: Expression,
     whenTrue: Command[],
-    whenFalse: Command[]
+    whenFalse: Command[] = [],
+    inline: Boolean = false,
```

Comment:
`Boolean` -> `boolean` (`Boolean` is the _class_ Boolean). Or you can just leave off the type.

Reply:
whoops 👍